include/netpacket/netlink.h

```diff
@@ -725,11 +719,9 @@ enum
   PREFIX_UNSPEC,
   PREFIX_ADDRESS,
   PREFIX_CACHEINFO,
-  __PREFIX_MAX
+  PREFIX_MAX
```

Comment:
Please note that [linux](https://github.com/torvalds/linux/blob/v6.8/include/uapi/linux/rtnetlink.h#L593)'s `PREFIX_MAX` is equal to `PREFIX_CACHEINFO`

Reply:
Fixed.